configurations/redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/AbstractRedisConfiguration.java

```diff
@@ -33,6 +33,7 @@
 
     private RedisURI uri;
     private List<RedisURI> uris = Collections.emptyList();
+    private String maxmemory = "";
```

Comment:
Is this only used in the embedded server?

Reply:
I would say rather than making this configurable, just put a sensible value in the server code. The embedded redis server shouldn't be used for anything more than a development environment and if it ends up someone needs to change this we can create an embedded configuration in the future.

Reply:
ok, done.